python/lsst/fgcmcal/fgcmBuildFromIsolatedStars.py

```diff
@@ -166,7 +166,7 @@ def setDefaults(self):
         source_selector = self.sourceSelector["science"]
         source_selector.setDefaults()
 
-        source_selector.doFlags = True
+        source_selector.doFlags = False
```

Comment:
Did you really mean to turn off (even the default science selector) flag filtering by default?

Reply:
Yes, this is intentional.  When running from the isolated stars the flags have already all been applied in the original isolated star association task, except for the local background flag.  What I have now may be redundant I realize, but it does work.